tests.md

```diff
@@ -1,30 +1,30 @@
 
-It consists of two tools: the Runner is the part that runs core's PHPUnit tests on a host and optionally [sends the information to the results page](https://make.wordpress.org/hosting/test-results/); and the Reporter which is the plugin that [works on the hosting page](https://make.wordpress.org/hosting/) and shows the results.
+It consists of two tools: the Runner is the part that runs core's PHPUnit tests on a host and optionally [sends the information to the results page](https://make.wordpress.org/hosting/test-results/); and the Reporter, which is the plugin that [works on the hosting page](https://make.wordpress.org/hosting/) and shows the results.
 
 ## What is it
 
 ### Runner
 
-Hosting companies can have several to millions of websites hosted with WordPress, so it's important to make sure their configuration is as compatible as possible with the software.
+Hosting companies can have millions of websites hosted with WordPress, so it's important to make sure their configuration is as compatible as possible with the software.
 
 To verify this compatibility, the WordPress Community provides a series of [PHPUnit](https://phpunit.de/) tests with which to check the operation of WordPress in any environment.
 
 ### Reporter
 
-The Runner tests generates a report with the test results related to a bot user (a hosting company), and this captures and displays those test results at the [Host Test Result](https://make.wordpress.org/hosting/test-results/) page.
+The Runner tests generate a report with the test results related to a bot user associated with a web host, and this captures and displays those test results at the [Host Test Result](https://make.wordpress.org/hosting/test-results/) page.
 
 ## Try the PHPUnit Test Runner
 
 ### What's the phpunit-test-runner
 
 The [phpunit-test-runner](https://github.com/WordPress/phpunit-test-runner) is a tool designed to make it easier for hosting companies to run the WordPress project's automated tests.
 
-There is a [whole documentation about this tool](https://make.wordpress.org/hosting/test-results-getting-started/). Also, if you want, you can make your test results appear in the [Host Test Results page](https://make.wordpress.org/hosting/test-results/) of WordPress.
+There is [full documentation about this tool](https://make.wordpress.org/hosting/test-results-getting-started/). Also, if you want, you can make your test results appear in the [Host Test Results page](https://make.wordpress.org/hosting/test-results/) of WordPress.
 
-The tool can be run manually or through an automated system like Travis. To see how it works and the purpose of this document, will be shown how to run the tests manually.
+The tool can be run manually or through an automated system like Travis. To see how it works, and the purpose of this document, is to show how to run the tests manually.
```

Comment:
The second sentence here is tripping up my brain. Is it saying something like "The purpose of this document is to show how it works and how to run the tests manually"?